index.html

```diff
@@ -1564,7 +1580,7 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
       Set <var>nfc</var>.[[\suspended]] to <code>true</code>.
     </li>
     <li>
-      For each pending <a>NFC.push</a>:
+      For each pending <a>NFCWriter.push</a>:
```

Comment:
I guess this is talking about making the NFCWriter object disappear, and we're doing this for each push pending on the NFCWriter object, right? (Since presumably this will be called for all the NFCWriter objects.)

We should probably have a little layering change in HTML's [unloading document cleanup steps](https://html.spec.whatwg.org/#unloading-document-cleanup-steps) to let all platform objects register "make disappear" steps. (And, going crazy with it, maybe we should have a JS API for this as well?)

(Doesn't need to be clarified in this PR)

Reply:
Added https://github.com/w3c/web-nfc/pull/178/commits/459a51a551b309278171f014f32ed9c2b39f4466 to clarify these things (I had previously missed the connection below, and seems like we don't need a layering change).